atticmatic/backends/borg.py

```diff
@@ -13,6 +13,7 @@
         (
             option('encryption_passphrase', required=False),
             option('compression', required=False),
+            option('umask', int, required=False),
```

Comment:
Should this be a string instead of an int, to support octal values properly? The example from the [Borg docs](https://borgbackup.readthedocs.org/en/stable/usage.html#borg-create) is `0077`. Taking an int config value and calling `str()` on it would result in just `77`, for instance. Not sure if this matters from Borg's perspective. I assume you've tried it out?